css/encore.css

```diff
@@ -6,24 +6,32 @@
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+  color: var(--txt-light);
 }
 
 html {
   box-sizing: inherit;
   font-size: 62.5%;
+  color: var(--txt-light);
 }
-
-body {
-  font-size: 20rem;
+ul,
+li {
+  list-style-type: none;
+}
+a {
+  text-decoration: none;
 }
 
 /* Css Variables */
 /* Defining all the main variables in root */
 :root {
```

Comment:
Hey, everything looks great and neat. One thing that I think is you can define all the main variables in root at top.

Reply:
Thanks . Done